src/main/kotlin/godot/entrygenerator/extension/StringExt.kt

```diff
@@ -1,5 +1,11 @@
 package godot.entrygenerator.extension
 
+fun String.camelToSnakeCase(): String {
```

Comment:
Note for the future, 
IIRC we have same funcgtion in entry generation, maybe have other methods that can be centralized.

Reply:
Jep we do have the same there but slightly different, so i opted for duplication. Does it really make sense to centralize such simple functions across projects?

Reply:
If so I'll change it in this PR. I prefer sooner than later :-)

Reply:
If it is not same impl maybe duplicate is better ^^